### DISCUSSION TITLE
Welcome to react-native-audio-remove Discussions!

### DISCUSSION THREAD

**Discussion by abhi3691** [abhi3691/react-native-audio-remove#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prakharjcis** [comment]
How to use local storage video?
(instead of https)

**Comment by abhi3691** [comment]
refer this github sample app code
https://github.com/abhi3691/videoprations.git


add   add bellow permissions for andorid 13   <uses-permission
android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
  <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"
    android:maxSdkVersion="32" />
  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"
/>
    <uses-permission android:name="android.permission.READ_MEDIA_AUDIO" />
